File: replication/src/deprecated-test/com/db4o/test/replication/SPCParent.java
/* Copyright (C) 2004 - 2005  db4objects Inc.  http://www.db4o.com */

package com.db4o.test.replication;


public class SPCParent {

	private SPCChild child;

	private String name;

	public SPCParent() {

	}

	public SPCParent(String name) {
		this.name = name;
	}

	public SPCParent(SPCChild child, String name) {
		this.child = child;
		this.name = name;
	}

	public SPCChild getChild() {
		return child;
	}

	public void setChild(SPCChild child) {
		this.child = child;
	}


	public String getName() {
		return name;
	}


	public void setName(String name) {
		this.name = name;
	}


	public String toString() {
		return "SPCParent{" +
				"child=" + child +
				", name='" + name + '\'' +
				'}';
	}
}

File: drs/drs.net/drs.tests/src/com/db4o/drs/test/SPCParent.cs
namespace com.db4o.drs.test
{
	public class SPCParent
	{
		private com.db4o.drs.test.SPCChild child;

		private string name;

		public SPCParent()
		{
		}

		public SPCParent(string name)
		{
			this.name = name;
		}

		public SPCParent(com.db4o.drs.test.SPCChild child, string name)
		{
			this.child = child;
			this.name = name;
		}

		public virtual com.db4o.drs.test.SPCChild GetChild()
		{
			return child;
		}

		public virtual void SetChild(com.db4o.drs.test.SPCChild child)
		{
			this.child = child;
		}

		public virtual string GetName()
		{
			return name;
		}

		public virtual void SetName(string name)
		{
			this.name = name;
		}

		public override string ToString()
		{
			return "SPCParent{" + "child=" + child + ", name='" + name + '\'' + '}';
		}
	}
}
